pvlib/clearsky.py

```diff
@@ -214,7 +211,14 @@ def lookup_linke_turbidity(time, latitude, longitude, filepath=None,
         np.around(_linearly_scale(longitude, -180, 180, 0, 4320))
         .astype(np.int64))
 
-    lts = linke_turbidity_table[latitude_index][longitude_index]
+    lt_h5_file = tables.open_file(filepath)
+    try:
+        lts = lt_h5_file.root.LinkeTurbidity[latitude_index, longitude_index, :]
+    except IndexError:
+        raise IndexError('Latitude should be between 90 and -90, '
```

Comment:
We should add a test that asserts that this exception is raised under invalid inputs.

Also, I confirmed that the code below does in fact execute the finally block despite a new exception being raised.

```python
try:
    raise IndexError
except IndexError:
    raise IndexError('blah')
finally:
    print('finally')
```

Reply:
I actually did write a test for that, but then realized it was [already covered](https://github.com/pvlib/pvlib-python/blob/master/pvlib/test/test_clearsky.py#L479).

For the `try except finally`, I was hoping `open_file` to come with a context manager, but it does not look like it is the case and fall back to the old school way to make sure the file get closed whatever happen.

Reply:
my bad, thanks for reminding me...